engine-server/prisma/migrations/20230627120327_add_email_to_integration_type/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterEnum
+ALTER TYPE "IntegrationType" ADD VALUE 'EMAIL';
```

Comment:
Where are we using an `IntegrationType` as `EMAIL`?

Reply:
This is to fix the migrations that were broken. This is from the original Poozle implementation. If I remove from DB values it can break the DB model.